src/sql.rs

```diff
@@ -618,7 +619,26 @@ async fn maybe_add_from_param(
         .query_map(
             query,
             paramsv![],
-            |row| row.get::<_, String>(0),
+            //|row| row.get::<_, String>(0), // <=== this crashes when non utf8 byte in column
+
+            // cs: maybe the following could be done even more simple than here?
+            |row| {
+                let value = match row.get_raw(0) {
+                    ValueRef::Text(t) => {
+                        // changed due to clippy complain
```

Comment:
What clippy says? Couldn't it be fixed without dropping the log?

Reply:
Value should be returned directly by `String::from_utf8_lossy(t).to_string()`
The `let col_str` should not be used.

Reply:
I have just uncommented your code (lines starting with `~`) and dropped `String::from_utf8_lossy(t).to_string()` that goes after them, and clippy is happy with this code.